candidatepair.go

```diff
@@ -28,6 +30,12 @@ type CandidatePair struct {
 	state                    CandidatePairState
 	nominated                bool
 	nominateOnBindingSuccess bool
+
+	// stats
+	statsMu              sync.RWMutex
```

Comment:
What do you think about making these atomic (if possible?)

Having another lock on such a critical path (inbound network traffic) might eventually be targeted for optimization.

Reply:
The STUN pings are usually once every few seconds. So, I think it should be fine.

But, I will look at atomics.